app/src/main/java/com/example/parkingsystem/util/Constants.kt

```diff
@@ -0,0 +1,5 @@
+package com.example.parkingsystem.util
+
+object Constants {
+    const val parkingSize=10
+}
```

Comment:
to avoid this error add an empty line at the end of the file

Reply:
Done 👍